contracts/jobs/JobsManager.sol

```diff
@@ -378,6 +382,9 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
         // Sender must be elected transcoder
         require(job.transcoderAddress == msg.sender);
 
+        uint256 blockNum = roundsManager().blockNum();
+        // Claim block + 1 must be within the last 256 blocks from the current block
+        require(blockNum < 256 || claim.claimBlock + 1 >= blockNum - 256);
```

Comment:
This check was originally in `JobLib.shouldVerifySegment`, but `JobLib` does not have access to RoundsManager to get the block number, so this check is moved out of `JobLib` and into `JobsManager`